cmd/zed/zed.d/zed.rc

```diff
@@ -15,7 +15,7 @@
 # Email will only be sent if ZED_EMAIL_ADDR is defined.
 # Disabled by default; uncomment to enable.
```

Comment:
```suggestion
# Enabled by default; comment to disable.
```

Reply:
@behlendorf thanks for catching the comment, I totally missed it.